packages/react-server/src/render/test/render.test.tsx

```diff
@@ -23,18 +22,16 @@ describe('createRender', () => {
   it('response contains "My cool app"', async () => {
     const myCoolApp = 'My cool app';
     const ctx = createMockContext();
-    setLogger(ctx, new Logger());
```

Comment:
cc/ @ismail-syed ended up just putting a fallback on `console` here cause it was make it hard to consume the `createRender` middleware as a standalone thing.

Reply:
Clever!